doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -188,7 +188,6 @@ Traceability to Reference Model.
 <!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
```

Comment:
may i suggest moving those to the bottom of the table (otherwise they wont be rendered correctly)